sdk/src/Core/AWSXRayRecorder.Core.csproj

```diff
@@ -49,6 +49,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' != 'net45' AND '$(TargetFramework)' != 'net462'">
     <PackageReference Include="Amazon.Lambda.SQSEvents" Version="2.1.0"/>
+    <PackageReference Include="Amazon.Lambda.Core" Version="2.8.0" />
```

Comment:
Is it possible that the Lambda .NET runtime environment includes an older version of this dependency, causing the LambdaTraceProvider to not have trace context?
In our testing with the Lambda Java runtime environment, we found that the user's application dependencies have lower priority than those bundled with Lambda itself, so a fallback logic is needed to continue using the legacy _X_AMZN_TRACE_ID.